src/types.ts

```diff
@@ -81,3 +81,9 @@ export interface ISlackAPIClient {
    */
   call(method: string, data: { [key: string]: unknown }): Promise<BaseResponse>;
 }
+
+export const EventTypes = {
+  FUNCTION_EXECUTED: "function_executed",
+} as const;
```

Comment:
nit: I feel like our `types.ts` files normally only contain TS types, but `EventTypes` is also an actual value. I feel like there is not really a perfect spot for it that already exists, but I was thinking we could:
- Put this in a separate file named something like `event-types.ts`
- Place it at the top of `dispatch-payload.ts` until we need it somewhere else
- Leave it here 🤷

Reply:
I'll move it to it's own file, but if it's ok I'll do it in the PR that adds the new types, it'll be a bit easier there w/ the changes I have around that area. By easier, I mean I won't have to deal with merge conflicts cause I'm lazeeee